frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -334,4 +334,79 @@ describe("scenarios > question > filter", () => {
       });
     });
   });
+
+  it.skip("should clear default filter value in native questions (metabase#13961)", () => {
+    const QUESTION_NAME = "13961";
+    const [CATEGORY_FILTER, ID_FILTER] = [
+      { name: "category", display_name: "Category", type: "dimension" },
+      { name: "prodid", display_name: "ProdId", type: "number" },
+    ];
+
+    cy.request("POST", "/api/card", {
+      name: QUESTION_NAME,
+      dataset_query: {
+        type: "native",
+        native: {
+          query:
+            "SELECT * FROM PRODUCTS WHERE 1=1 AND {{category}} [[AND ID={{prodid}}]]",
+          "template-tags": {
+            [CATEGORY_FILTER.name]: {
+              id: "00315d5e-4a41-99da-1a41-e5254dacff9d",
+              name: CATEGORY_FILTER.name,
+              "display-name": CATEGORY_FILTER.display_name,
+              type: CATEGORY_FILTER.type,
+              default: "Doohickey",
+              dimension: ["field-id", PRODUCTS.CATEGORY],
+              "widget-type": "category",
+            },
+            [ID_FILTER.name]: {
+              id: "4775bccc-e82a-4069-fc6b-2acc90aadb8b",
+              name: ID_FILTER.name,
+              "display-name": ID_FILTER.display_name,
+              type: ID_FILTER.type,
+              default: null,
+            },
+          },
+        },
+        database: 1,
+      },
+      display: "table",
+      visualization_settings: {},
+    }).then(({ body: { id: QUESTION_ID } }) => {
+      cy.visit(`/question/${QUESTION_ID}`);
+
+      cy.findByText(QUESTION_NAME);
+      cy.findAllByText("Small Marble Shoes"); // Product ID 2, Doohickey
+
+      cy.location("search").should("eq", "?category=Doohickey");
+
+      // Remove default filter (category)
+      cy.get("fieldset .Icon-close").click();
+
+      cy.get(".Icon-play")
+        .first()
+        .should("be.visible")
+        .as("rerunQuestion");
+
+      cy.get("@rerunQuestion").click();
+      cy.url().should("not.include", "?category=Doohickey");
+
+      // Add value `1` to the ID filter
+      cy.findByPlaceholderText(ID_FILTER.display_name).type("1");
+
+      cy.get("@rerunQuestion").click();
+
+      cy.log("**--Reported tested and failing on v0.34.3 through v0.37.3--**");
+      cy.log("**URL is correct at this point, but there are no results**");
+      cy.location("search").should("eq", `?${ID_FILTER.name}=1`);
+      cy.findByText("Rustic Paper Wallet"); // Product ID 1, Gizmo
+
+      cy.reload();
+      cy.findByText(QUESTION_NAME);
+      cy.log(
+        "**After reload, even the URL fails because it includes the default category filter again**",
+      );
+      cy.location("search").should("eq", `?${ID_FILTER.name}=1`);
```

Comment:
But it should automatically add default value to non-included parameters, and it changes the URL, so that is correct.
```suggestion
```

Reply:
Oh, because we didn't save the question again - that's why?

Reply:
It's the same problem as dashboards, where it's not possible to tell URL to not include default values.
This issue is slightly different, since it shows as if the default value is not included, but the query still includes it.
But it should still be possible to just define one parameter, and then the rest are set to the defaults - otherwise we would break backward compatibility.